examples/paper.md

```diff
@@ -147,6 +144,8 @@ affiliations:
    index: 4
  - name: Google Inc.
    index: 5
+ - name: Université de Lyon
```

Comment:
If going with the short version, it should be "École Normale Supérieure de Lyon" for me

Thanks!